### PR TITLE
Use Paket (magic mode) and publish Nuget package

### PR DESCRIPTION
**Pull request by alfonsogarciacaro**
See https://github.com/fable-elmish/elmish/pull/93

### CODE REVIEW

**Comment by et1975** [comment]
I understand that the project file now doubles as the nuget template, will fable handle the project OK if I add nuget metadata?

**Comment by alfonsogarciacaro** [comment]
[Fable parsing of .fsproj files](https://github.com/fable-compiler/Fable/blob/e8e72ffcfe09c6f15c5c4b49db57dac4286064ff/src/dotnet/Fable.Tools/ProjectCracker.fs#L119-L160) is very simple. Basically it only cares about Compile/Reference/ProjectReference tags and ignores the rest :) If needed, we could also use Enrico's library to interpret the project file with MSBuild.

**Comment by et1975** [comment]
"ignores the rest" works for me :)

File: build.cmd [hunk 1]
```diff
@@ -1,11 +1,6 @@
 @echo off
 cls
 
-.paket\paket.bootstrapper.exe
```
Comment (et1975) [inline build.cmd:4]: We don't like bootstrapper?

Comment (alfonsogarciacaro) [reply build.cmd:4]: `paket.exe` is now `paket.bootstrapper.exe` in disguise ;) That's the magic mode, when you send a command to paket.exe, the bootrapper will download latest Paket version to a temporary location and run it. So basically it's the same as with the bootstrapper (you always have the latest version) while for the user there's only one tiny executable that can be also committed to the repo :+1:

**Pull request merged**